apps/extension/src/routes/popup/popup-layout.tsx

```diff
@@ -12,7 +12,7 @@ import { Outlet } from 'react-router-dom';
  * then removing the hard-coded width from `globals.css`.
  */
 export const PopupLayout = () => (
-  <div className='flex min-h-full flex-col bg-card-radial'>
+  <div className='flex grow flex-col bg-card-radial'>
```

Comment:
Not sure why `height: 100%` doesn't fill a flexbox, but apparently it doesn't. So I had to use `grow` instead.